Makefile

```diff
@@ -240,6 +244,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure --without-snapshot $(CONFIG_FLAGS)
```

Comment:
Thanks for pointing that out. It wasn't in Jenkins so that why I didn't put it here (I just copied what was in Jenkins).
I will try to add this to this PR.